python/lib/evaluations/dmod/evaluations/threshold/retriever.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+import typing
+import abc
+
+import pandas
+
+from .. import specification
+from .. import backends
+
+
+class ThresholdRetriever(abc.ABC):
```

Comment:
_ThresholdRetriever_, _DataRetriever_, and _CrosswalkRetriever_ appear to basically be the same abstraction.  There are some class methods that appear in some, but I'm not sure there are really needed at the abstract level (even then, for two of the three, there is overlap).   Might be worth refactoring both the abstractions and/or the packaging layout.

Reply:
While `DataRetriever` (the class getting evaluation input) and `CrosswalkRetriever` (the class getting/interpreting the crosswalk) have their own little quirks/assumptions, `ThresholdRetriever` really is a good intersection between all of them. I'm gonna see what I can do to distill all of that while I'm looking at it. Good find.